Essay_Analysis_Tool/DiffViewerForm.cs

```diff
@@ -95,33 +95,41 @@ private void FastColoredTextBox_VisibleRangeChanged(object sender, EventArgs e)
                 return;
 
             var vPos = (sender as FastColoredTextBox).VerticalScroll.Value;
+            var hPos = (sender as FastColoredTextBox).HorizontalScroll.Value;
             var curLine = (sender as FastColoredTextBox).Selection.Start.iLine;
 
             if (sender == fctb2)
-                UpdateScroll(fctb1, vPos, curLine);
+                UpdateScroll(fctb1, vPos, hPos, curLine);
             else
-                UpdateScroll(fctb2, vPos, curLine);
+                UpdateScroll(fctb2, vPos, hPos, curLine);
 
             fctb1.Refresh();
             fctb2.Refresh();
```

Comment:
If i remove the two `fctb.Refresh()` method the line number section become less laggy but it updates the two FCTB instance less often so when scrolling the text content jump instead of smoothly move left/right/top/down in the screen.

Reply:
Sorry for the lack of responses out of me, I've been a bit busy. I will try and take a look at this commit on the 26th to see if there is a fix for the UI latency.

Reply:
Oh, not a problem for me, You sad you are a student too, so no worries :) Still no response to my [question](https://github.com/PavelTorgashov/FastColoredTextBox/issues/70#issuecomment-533865160).

Reply:
Yeah, Pavel doesn't really maintain that repository anymore since this component is very old and there are several others who have forked the component and customized it (which is more than likely the original intended purpose). So we will more than likely have to resolve this issue ourselves. Luckily I am one of the people that have forked the repo, you can view my fork here: https://github.com/Hexman768/FastColoredTextBox

Reply:
All right, i forked it too and started to refactor some of it. But i will take a look at Yours as soon as possible.

Reply:
Didn't want you to think I left or anything, just taking a little break from this project until I can get to a point where I can actively work on this project, in the mean time I was wanting to talk to you about a possible translation project for Notepad#

Reply:
Hi!

No problem, i am busy too cos' the school. Yeah, i think i can do a hungarian translation.

Reply:
That would be amazing. However, I would like to speak with you more on this subject. I will send you an email in the next couple of days laying out the plan for the translation project.

Reply:
Have we found a solution for this issue yet? I just gave it a go and was able to reproduce the issue on the latest commit on your feature/diffmerge branch.

Reply:
Tell you what, it isn't that big of a deal to me to NOT have horizontal scrolling (in fact, there may be more of an argument AGAINST having it) so just remove that functionality and we will call this commit good to go and get it merged that way we can close the issue and move on.